src/index.html

```diff
@@ -3,11 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
```

Comment:
14. [checklist] place every attribute on the new line with 2-space indentation like this:

```suggestion
<meta
  name="viewport"
  content="width=device-width, user-scalable=no...
>
```